### PR TITLE
TWEAK: Titles renames + small folder reorganisation

### PR DESCRIPTION
**Pull request by PurpleHato**

### CODE REVIEW

File: data/Music/Mario & Luigi Series/Mario & Luigi  Dream Team Bros/Never Let Up.meta [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Mario & Luigi: Dream Team Bros. - Never Let Up
```
Comment (TheSoundDefense) [inline data/Music/Mario & Luigi Series/Mario & Luigi  Dream Team Bros/Never Let Up.meta:1]: This should just be "Mario & Luigi: Dream Team".

Comment (TheSoundDefense) [inline data/Music/Mario & Luigi Series/Mario & Luigi  Dream Team Bros/Never Let Up.meta:1]: This should just be "Mario & Luigi: Dream Team". Could you also change the directory?
```suggestion
Mario & Luigi: Dream Team - Never Let Up
```

File: data/Music/Mega Man Series/Mega Man 3/Shadow.meta [hunk 2]
```diff
@@ -1,3 +1,3 @@
-Mega Man 3 - Shadow Man
+Mega Man III - Shadow Man
```
Comment (TheSoundDefense) [inline data/Music/Mega Man Series/Mega Man 3/Shadow.meta:1]: Roman numerals are not appropriate for the classic Mega Man series. Those games use Arabic numerals. There is actually a separate line of Mega Man games on the Game Boy that use Roman numerals, and this change may cause some confusion.

Only use Roman numerals if that is how the game is officially stylized.

```suggestion
Mega Man 3 - Shadow Man
```

File: data/Music/Mega Man Series/Mega Man 3/Title.meta [hunk 3]
```diff
@@ -1,3 +1,3 @@
-Mega Man 3 - Title
+Mega Man III - Title
```
Comment (TheSoundDefense) [inline data/Music/Mega Man Series/Mega Man 3/Title.meta:1]: Roman numerals are not appropriate for the classic Mega Man series.

```suggestion
Mega Man 3 - Title
```

File: data/Music/Mega Man Series/Mega Man 6/Final Boss.meta [hunk 4]
```diff
@@ -1,3 +1,3 @@
-Mega Man 6 - Final Boss
+Mega Man VI - Final Boss
```
Comment (TheSoundDefense) [inline data/Music/Mega Man Series/Mega Man 6/Final Boss.meta:1]: Roman numerals are not appropriate for the classic Mega Man series.

```suggestion
Mega Man 6 - Final Boss
```

File: data/Music/Mega Man Series/Mega Man 9/Final Boss.meta [hunk 5]
```diff
@@ -1,3 +1,3 @@
-Mega Man 9 - Final Boss
+Mega Man IX - Final Boss
```
Comment (TheSoundDefense) [inline data/Music/Mega Man Series/Mega Man 9/Final Boss.meta:1]: Roman numerals are not appropriate for the classic Mega Man series.
```suggestion
Mega Man 9 - Final Boss
```

File: data/Music/Mega Man Series/Mega Man 9/Galaxy.meta [hunk 6]
```diff
@@ -1,3 +1,3 @@
-Mega Man 9 - Galaxy Man
+Mega Man IX - Galaxy Man
```
Comment (TheSoundDefense) [inline data/Music/Mega Man Series/Mega Man 9/Galaxy.meta:1]: Roman numerals are not appropriate for the classic Mega Man series.
```suggestion
Mega Man 9 - Galaxy Man
```

File: data/Music/Mega Man Series/Mega Man 9/Hornet.meta [hunk 7]
```diff
@@ -1,3 +1,3 @@
-Mega Man 9 - Hornet Man
+Mega Man IX - Hornet Man
```
Comment (TheSoundDefense) [inline data/Music/Mega Man Series/Mega Man 9/Hornet.meta:1]: Roman numerals are not appropriate for the classic Mega Man series.
```suggestion
Mega Man 9 - Hornet Man
```

File: data/Music/Mega Man Series/Mega Man 9/We are the Robots.meta [hunk 8]
```diff
@@ -1,3 +1,3 @@
-Mega Man 9 - We Are The Robots
+Mega Man IX - We Are The Robots
```
Comment (TheSoundDefense) [inline data/Music/Mega Man Series/Mega Man 9/We are the Robots.meta:1]: Roman numerals are not appropriate for the classic Mega Man series.
```suggestion
Mega Man 9 - We Are the Robots
```

File: data/Music/Metroid Series/Metroid Fusion/Sector 1.meta [hunk 9]
```diff
@@ -1,3 +1,3 @@
-Metroid Fusion - Sector 1
+Metroid: Fusion - Sector 1
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Fusion/Sector 1.meta:1]: The official name of this game is "Metroid Fusion" with no colon.
```suggestion
Metroid Fusion - Sector 1
```

File: data/Music/Metroid Series/Metroid Fusion/Sector 2.meta [hunk 10]
```diff
@@ -1,3 +1,3 @@
-Metroid Fusion - Sector 2
+Metroid: Fusion - Sector 2
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Fusion/Sector 2.meta:1]: The official name of this game is "Metroid Fusion" with no colon.
```suggestion
Metroid Fusion - Sector 2
```

File: data/Music/Metroid Series/Metroid Fusion/Sector 3.meta [hunk 11]
```diff
@@ -1,3 +1,3 @@
-Metroid Fusion - Sector 3
+Metroid: Fusion - Sector 3
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Fusion/Sector 3.meta:1]: The official name of this game is "Metroid Fusion" with no colon.
```suggestion
Metroid Fusion - Sector 3
```

File: data/Music/Metroid Series/Metroid Fusion/Sector 4.meta [hunk 12]
```diff
@@ -1,3 +1,3 @@
-Metroid Fusion - Sector 4
+Metroid: Fusion - Sector 4
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Fusion/Sector 4.meta:1]: The official name of this game is "Metroid Fusion" with no colon.
```suggestion
Metroid Fusion - Sector 4
```

File: data/Music/Metroid Series/Metroid Fusion/Sector 5.meta [hunk 13]
```diff
@@ -1,3 +1,3 @@
-Metroid Fusion - Sector 5
+Metroid: Fusion - Sector 5
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Fusion/Sector 5.meta:1]: The official name of this game is "Metroid Fusion" with no colon.
```suggestion
Metroid Fusion - Sector 5
```

File: data/Music/Metroid Series/Metroid Fusion/Serris.meta [hunk 14]
```diff
@@ -1,3 +1,3 @@
-Metroid Fusion - Serris
+Metroid: Fusion - Serris
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Fusion/Serris.meta:1]: The official name of this game is "Metroid Fusion" with no colon.
```suggestion
Metroid Fusion - Serris
```

File: data/Music/Metroid Series/Metroid Prime Hunters/Alinos Multiplayer.meta [hunk 15]
```diff
@@ -1,3 +1,3 @@
-Metroid Prime Hunters - Alinos Multiplayer
+Metroid Prime: Hunters - Alinos Multiplayer
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Prime Hunters/Alinos Multiplayer.meta:1]: The official name of the game is "Metroid Prime Hunters".
```suggestion
Metroid Prime Hunters - Alinos Multiplayer
```

File: data/Music/Metroid Series/Metroid Prime Hunters/Arcterra Multiplayer.meta [hunk 16]
```diff
@@ -1,3 +1,3 @@
-Metroid Prime Hunters - Arcterra Multiplayer
+Metroid Prime: Hunters - Arcterra Multiplayer
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Prime Hunters/Arcterra Multiplayer.meta:1]: The official name of the game is "Metroid Prime Hunters".
```suggestion
Metroid Prime Hunters - Arcterra Multiplayer
```

File: data/Music/Metroid Series/Metroid Prime Hunters/Arcterras Depths.meta [hunk 17]
```diff
@@ -1,3 +1,3 @@
-Metroid Prime Hunters - Arcterras Depths
+Metroid Prime: Hunters - Arcterras Depths
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Prime Hunters/Arcterras Depths.meta:1]: The official name of the game is "Metroid Prime Hunters".
```suggestion
Metroid Prime Hunters - Arcterras Depths
```

File: data/Music/Metroid Series/Metroid Prime Hunters/Celestial Archives.meta [hunk 18]
```diff
@@ -1,3 +1,3 @@
-Metroid Prime Hunters - Celestial Archives
+Metroid Prime: Hunters - Celestial Archives
```
Comment (TheSoundDefense) [inline data/Music/Metroid Series/Metroid Prime Hunters/Celestial Archives.meta:1]: The official name of the game is "Metroid Prime Hunters".
```suggestion
Metroid Prime Hunters - Celestial Archives
```

File: data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/Sky Sanctuary.meta [hunk 19]
```diff
@@ -1,4 +1,4 @@
-Sonic and Knuckles - Sky Sanctuary
+Sonic the Hedgehog 3 & Knuckless - Sky Sanctuary
```
Comment (TheSoundDefense) [inline data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 3/Sky Sanctuary.meta:1]:
```suggestion
Sonic the Hedgehog 3 & Knuckles - Sky Sanctuary
```

**Review by TheSoundDefense — COMMENTED** [review]
Cleaning up the repo is always appreciated, but make sure you are using the correct game names before making corrections. Thanks!

**Comment by DaruniasJoy** [comment]
changed what was marked incorrect.

**Pull request merged**